src/ZendeskApi.Acceptance/SatisfactionRatingSteps.cs

```diff
@@ -22,51 +22,57 @@ public class SatisfactionRatingSteps
         [BeforeScenario]
         public void BeforeScenario()
         {
-            _customerClient = new ZendeskClient(new Uri(ConfigurationManager.AppSettings["zendeskhost"]),
-                new ZendeskDefaultConfiguration(ConfigurationManager.AppSettings["zendeskenduserusername"], ConfigurationManager.AppSettings["zendesktoken"]));
+            _customerClient = new ZendeskClient(Host, new ZendeskDefaultConfiguration(Enduser, Token));
 
-            _supportUserClient = new ZendeskClient(new Uri(ConfigurationManager.AppSettings["zendeskhost"]),
-                 new ZendeskDefaultConfiguration(ConfigurationManager.AppSettings["zendeskusername"], ConfigurationManager.AppSettings["zendesktoken"]));
+            _supportUserClient = new ZendeskClient(Host, new ZendeskDefaultConfiguration(Username, Token));
         }
 
         [Given(@"a ticket in Zendesk with the subject '(.*)' and description '(.*)'")]
         public void GivenATicketInZendeskWithTheSubjectAndDescription(string subject, string description)
         {
-            var requesterId =
-                _supportUserClient.Search.Find<User>(new ZendeskQuery<User>().WithCustomFilter("email",
-                    ConfigurationManager.AppSettings["zendeskenduserusername"], FilterOperator.Equals)).Results.First().Id;
+            var query = new ZendeskQuery<User>().WithCustomFilter("email",
+                ConfigurationManager.AppSettings["zendeskenduserusername"], FilterOperator.Equals);
```

Comment:
Consider using your new Enduser string here instead of calling the ConfigurationManager directly